pkg/agent/attestor/node/node.go

```diff
@@ -147,8 +146,8 @@ func IsSVIDExpired(svid []*x509.Certificate, timeNow func() time.Time) bool {
 }
 
 func (a *attestor) loadBundle() (*bundleutil.Bundle, error) {
-	bundle, err := manager.ReadBundle(a.c.BundleCachePath)
-	if errors.Is(err, manager.ErrNotCached) {
+	bundle, err := a.c.Storage.LoadBundle()
+	if errors.Is(err, storage.ErrNotCached) {
```

Comment:
If the agent starts after upgrade and the bundle is in a separate file, will this return `storage.ErrNotCached`?

Reply:
If the agent starts after upgrade, the "legacy" bundle is loaded into the new storage file when storage is initialized. This should return the "legacy" bundle.